microsite/src/main/mdoc/streams.md

```diff
@@ -48,6 +49,38 @@ val program: Future[Unit] = for {
 Await.ready(program, 5.seconds)
 ```
 
+### Pekko Streams _(neotypes-akka-stream)_
+
+```scala mdoc:compile-only
+import org.apache.pekko.NotUsed
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.stream.scaladsl.{Sink, Source}
+import neotypes.{GraphDatabase, StreamDriver}
+import neotypes.akkastreams.AkkaStream
+import neotypes.akkastreams.implicits._ // Brings the implicit Stream[AkkaStream] instance into the scope.
```

Comment:
`pekkostreams` and `PekkoStram` :)

Reply:
Fixup: 9d897cc674f5cb9095348ea81bba9fd3ce0dd63e